PMSCore/ReportManager.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using PMSCore;
+
+namespace PMSCore
+{
+    /// <summary>
+    /// This class is used to handle and create reports.
+    /// </summary>
+    public class ReportManager
```

Comment:
I'm afraid report manager should be a singleton like ProjectManager or TaskManager so only one instance exists in the entire execution

Reply:
I have now updated the ReportManager.cs and also ReportManagerTest.cs so that it's a singleton. Can this be reviewed, please?